File: mathvar-go/datavar.go
// datavar.go
package main

import "fmt"

func main() {
    a := 5
    b := 3
    sum := a + b
    diff := a - b
    prod := a * b
    quot := a / b

    fmt.Println("5 + 3 =", sum)
    fmt.Println("5 - 3 =", diff)
    fmt.Println("5 * 3 =", prod)
    fmt.Println("5 / 3 =", quot)
}

File: datavar.erl
% datavar.erl
-module(datavar).
-export([main/0]).

main() ->
    A = 5,
    B = 3,
    Sum = A + B,
    Diff = A - B,
    Prod = A * B,
    Quot = A div B,
    io:format("5 + 3 = ~p~n", [Sum]),
    io:format("5 - 3 = ~p~n", [Diff]),
    io:format("5 * 3 = ~p~n", [Prod]),
    io:format("5 / 3 = ~p~n", [Quot]).
